pkgs/tools/networking/netbird/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib, nixosTests, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "netbird";
+  version = "0.8.6";
+
+  src = fetchFromGitHub {
+    owner = "netbirdio";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-POf/IEP1WRLJ5AMsvH76j97wJcQ2/dT91TyP0x/qnsg=";
+  };
+
+  vendorSha256 = "sha256-KtRQwrCBsOX7Jk9mKdDNOD7zfssADfBXCO1RPZbp5Aw=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  subPackages = [ "client" ];
```

Comment:
Maybe we should also package the other executables, especially the UI?

Reply:
Done.